[2023-12-20 16:31:27,692: INFO: main: Welcome to our custom log]
[2023-12-27 12:11:21,118: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-27 12:11:21,121: INFO: common: yaml file: params.yaml loaded successfully]
[2023-12-27 12:11:21,123: INFO: common: created directory at: artifacts]
[2023-12-27 12:23:54,542: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-27 12:23:54,544: INFO: common: yaml file: params.yaml loaded successfully]
[2023-12-27 12:23:54,545: INFO: common: created directory at: artifacts]
[2023-12-27 12:23:54,547: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-27 12:23:54,548: INFO: 3525658959: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2023-12-27 12:25:48,691: INFO: common: yaml file: config\config.yaml loaded successfully]
[2023-12-27 12:25:48,693: INFO: common: yaml file: params.yaml loaded successfully]
[2023-12-27 12:25:48,695: INFO: common: created directory at: artifacts]
[2023-12-27 12:25:48,696: INFO: common: created directory at: artifacts/data_ingestion]
[2023-12-27 12:25:48,697: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2023-12-27 12:28:38,077: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-28 14:46:12,615: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-02-28 14:46:12,617: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-28 14:46:12,618: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 14:46:12,618: INFO: common: created directory at: artifacts]
[2024-02-28 14:46:12,618: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-28 14:46:12,619: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-28 14:46:14,745: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-28 14:46:14,746: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip']
Traceback (most recent call last):
  File "main.py", line 8, in <module>
    obj.main()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 15, in main
    data_ingestion.extract_zip_file()
  File "d:\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 43, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\zipfile.py", line 1251, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip'
[2024-02-28 14:52:58,888: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-02-28 14:52:58,889: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-28 14:52:58,890: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 14:52:58,890: INFO: common: created directory at: artifacts]
[2024-02-28 14:52:58,890: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-28 14:52:58,890: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-28 14:53:01,014: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-28 14:53:01,014: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip']
Traceback (most recent call last):
  File "main.py", line 8, in <module>
    obj.main()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 15, in main
    data_ingestion.extract_zip_file()
  File "d:\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 43, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\zipfile.py", line 1251, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip'
[2024-02-28 14:57:12,452: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-02-28 14:57:12,454: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-02-28 14:57:12,454: INFO: common: yaml file: params.yaml loaded successfully]
[2024-02-28 14:57:12,455: INFO: common: created directory at: artifacts]
[2024-02-28 14:57:12,455: INFO: common: created directory at: artifacts/data_ingestion]
[2024-02-28 14:57:12,455: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-28 14:57:15,005: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-02-28 14:57:15,401: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-03-20 12:51:25,643: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 12:51:25,650: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 12:51:25,654: INFO: common: created directory at: artifacts]
[2024-03-20 13:12:09,350: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 13:12:09,353: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 13:12:09,355: INFO: common: created directory at: artifacts]
[2024-03-20 13:14:33,159: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 13:14:33,162: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 13:14:33,164: INFO: common: created directory at: artifacts]
[2024-03-20 13:14:39,245: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 13:14:39,249: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 13:14:39,251: INFO: common: created directory at: artifacts]
[2024-03-20 13:18:05,515: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 13:18:05,518: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 13:18:05,520: INFO: common: created directory at: artifacts]
[2024-03-20 13:18:05,521: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-20 13:19:51,651: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-20 21:21:19,094: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-03-20 21:21:19,097: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 21:21:19,098: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 21:21:19,098: INFO: common: created directory at: artifacts]
[2024-03-20 21:21:19,099: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-20 21:21:19,099: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:21:20,792: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:21:21,237: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-03-20 21:21:21,237: INFO: main: *******************]
[2024-03-20 21:21:21,237: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-20 21:21:21,239: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 21:21:21,241: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 21:21:21,241: INFO: common: created directory at: artifacts]
[2024-03-20 21:21:21,241: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-20 21:21:21,620: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-20 21:21:21,818: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-03-20 21:48:37,671: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-03-20 21:48:37,674: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 21:48:37,676: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 21:48:37,676: INFO: common: created directory at: artifacts]
[2024-03-20 21:48:37,676: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-20 21:48:37,676: INFO: data_ingestion: Downloading data from https://drive.usercontent.google.com/download?id=1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy&export=download&authuser=0&confirm=t&uuid=1f1e2cde-1749-4d68-9e0f-0957fa3da224&at=APZUnTVWGZbuQ7rSqNHPkvut1rcJ%3A1710951134835 into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:48:38,426: INFO: data_ingestion: Downloaded data from https://drive.usercontent.google.com/download?id=1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy&export=download&authuser=0&confirm=t&uuid=1f1e2cde-1749-4d68-9e0f-0957fa3da224&at=APZUnTVWGZbuQ7rSqNHPkvut1rcJ%3A1710951134835 into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:48:38,788: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-03-20 21:48:38,789: INFO: main: *******************]
[2024-03-20 21:48:38,789: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-20 21:48:38,791: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 21:48:38,792: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 21:48:38,793: INFO: common: created directory at: artifacts]
[2024-03-20 21:48:38,793: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-20 21:48:39,191: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-20 21:48:39,395: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-03-20 21:51:17,419: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-03-20 21:51:17,422: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 21:51:17,424: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 21:51:17,424: INFO: common: created directory at: artifacts]
[2024-03-20 21:51:17,424: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-20 21:51:17,425: INFO: data_ingestion: Downloading data from https://drive.usercontent.google.com/download?id=1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy&export=download&authuser=0&confirm=t&uuid=1f1e2cde-1749-4d68-9e0f-0957fa3da224&at=APZUnTVWGZbuQ7rSqNHPkvut1rcJ%3A1710951134835 into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:51:18,053: INFO: data_ingestion: Downloaded data from https://drive.usercontent.google.com/download?id=1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy&export=download&authuser=0&confirm=t&uuid=1f1e2cde-1749-4d68-9e0f-0957fa3da224&at=APZUnTVWGZbuQ7rSqNHPkvut1rcJ%3A1710951134835 into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:51:18,053: ERROR: main: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip']
Traceback (most recent call last):
  File "main.py", line 9, in <module>
    obj.main()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 15, in main
    data_ingestion.extract_zip_file()
  File "d:\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 43, in extract_zip_file
    with zipfile.ZipFile(self.config.local_data_file, 'r') as zip_ref:
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\zipfile.py", line 1251, in __init__
    self.fp = io.open(file, filemode)
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts/data_ingestion/data.zip'
[2024-03-20 21:52:38,451: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-03-20 21:52:38,454: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 21:52:38,456: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 21:52:38,456: INFO: common: created directory at: artifacts]
[2024-03-20 21:52:38,456: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-20 21:52:38,456: INFO: data_ingestion: Downloading data from https://drive.usercontent.google.com/download?id=1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy&export=download&authuser=0&confirm=t&uuid=1f1e2cde-1749-4d68-9e0f-0957fa3da224&at=APZUnTVWGZbuQ7rSqNHPkvut1rcJ%3A1710951134835 into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:52:39,092: ERROR: main: Failed to retrieve file url:

	Cannot retrieve the public link of the file. You may need to change
	the permission to 'Anyone with the link', or have had many accesses.
	Check FAQ in https://github.com/wkentaro/gdown?tab=readme-ov-file#faq.

You may still be able to access the file from the browser:

	https://drive.google.com/uc?/export=download&id=drive.usercontent.google.com

but Gdown can't. Please check connections and permissions.]
Traceback (most recent call last):
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\gdown\download.py", line 253, in download
    url = get_url_from_gdrive_confirmation(res.text)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\gdown\download.py", line 54, in get_url_from_gdrive_confirmation
    raise FileURLRetrievalError(
gdown.exceptions.FileURLRetrievalError: Cannot retrieve the public link of the file. You may need to change the permission to 'Anyone with the link', or have had many accesses. Check FAQ in https://github.com/wkentaro/gdown?tab=readme-ov-file#faq.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "main.py", line 9, in <module>
    obj.main()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 14, in main
    data_ingestion.download_file()
  File "d:\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 31, in download_file
    raise e
  File "d:\kidney-disease-classification\src\cnnClassifier\components\data_ingestion.py", line 26, in download_file
    gdown.download(prefix+file_id,zip_download_dir)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\gdown\download.py", line 264, in download
    raise FileURLRetrievalError(message)
gdown.exceptions.FileURLRetrievalError: Failed to retrieve file url:

	Cannot retrieve the public link of the file. You may need to change
	the permission to 'Anyone with the link', or have had many accesses.
	Check FAQ in https://github.com/wkentaro/gdown?tab=readme-ov-file#faq.

You may still be able to access the file from the browser:

	https://drive.google.com/uc?/export=download&id=drive.usercontent.google.com

but Gdown can't. Please check connections and permissions.
[2024-03-20 21:53:33,149: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-03-20 21:53:33,151: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 21:53:33,153: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 21:53:33,153: INFO: common: created directory at: artifacts]
[2024-03-20 21:53:33,153: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-20 21:53:33,153: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:53:40,686: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-20 21:53:41,063: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-03-20 21:53:41,063: INFO: main: *******************]
[2024-03-20 21:53:41,064: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-20 21:53:41,066: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 21:53:41,068: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 21:53:41,068: INFO: common: created directory at: artifacts]
[2024-03-20 21:53:41,068: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-20 21:53:41,450: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-20 21:53:41,677: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-03-20 22:07:00,636: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 22:07:00,644: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 22:07:00,646: INFO: common: created directory at: artifacts]
[2024-03-20 22:09:24,844: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 22:09:24,847: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 22:09:24,849: INFO: common: created directory at: artifacts]
[2024-03-20 22:10:37,624: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 22:10:37,627: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 22:10:37,629: INFO: common: created directory at: artifacts]
[2024-03-20 22:13:05,373: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 22:13:05,377: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 22:13:05,379: INFO: common: created directory at: artifacts]
[2024-03-20 22:19:34,768: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 22:19:34,771: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 22:19:34,773: INFO: common: created directory at: artifacts]
[2024-03-20 22:19:34,775: INFO: common: created directory at: artifacts\training]
[2024-03-20 23:11:46,271: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-20 23:11:46,275: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-20 23:11:46,276: INFO: common: created directory at: artifacts]
[2024-03-20 23:11:46,278: INFO: common: created directory at: artifacts\training]
[2024-03-21 00:31:54,057: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-03-21 00:31:54,060: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-21 00:31:54,062: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 00:31:54,062: INFO: common: created directory at: artifacts]
[2024-03-21 00:31:54,063: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-21 00:31:54,064: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-21 00:31:59,632: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-21 00:32:00,048: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-03-21 00:32:00,048: INFO: main: *******************]
[2024-03-21 00:32:00,048: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-03-21 00:32:00,051: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-21 00:32:00,053: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 00:32:00,053: INFO: common: created directory at: artifacts]
[2024-03-21 00:32:00,053: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-21 00:32:00,439: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-21 00:32:00,642: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-03-21 00:32:00,642: INFO: main: *******************]
[2024-03-21 00:32:00,643: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-03-21 00:32:00,646: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-21 00:32:00,647: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 00:32:00,648: INFO: common: created directory at: artifacts]
[2024-03-21 00:32:00,648: INFO: common: created directory at: artifacts\training]
[2024-03-21 00:38:45,956: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-03-21 10:13:23,413: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-21 10:13:23,421: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 10:13:23,424: INFO: common: created directory at: artifacts]
[2024-03-21 10:13:43,998: INFO: common: json file saved at: scores.json]
[2024-03-21 10:13:51,899: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-03-21 10:13:53,026: INFO: builder_impl: Assets written to: C:\Users\vaish\AppData\Local\Temp\tmp7j0ogdhj\model\data\model\assets]
[2024-03-21 11:06:58,561: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-21 11:06:58,565: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 11:06:58,566: INFO: common: created directory at: artifacts]
[2024-03-21 11:06:58,568: INFO: common: created directory at: artifacts\training]
[2024-03-21 12:53:39,946: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-21 12:53:39,950: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-21 12:53:39,951: INFO: common: created directory at: artifacts]
[2024-03-21 12:53:39,954: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-21 12:53:39,955: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-31 15:16:40,149: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-31 15:16:40,152: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-31 15:16:40,154: INFO: common: created directory at: artifacts]
[2024-03-31 15:16:40,155: INFO: common: created directory at: artifacts/data_ingestion]
[2024-03-31 15:16:40,157: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-31 15:19:30,915: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-03-31 15:39:30,549: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-31 15:39:30,552: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-31 15:39:30,554: INFO: common: created directory at: artifacts]
[2024-03-31 15:39:30,556: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-31 15:39:30,981: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-31 16:08:29,867: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-31 16:08:29,870: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-31 16:08:29,872: INFO: common: created directory at: artifacts]
[2024-03-31 16:08:29,873: INFO: common: created directory at: artifacts\training]
[2024-03-31 17:54:21,287: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-31 17:54:21,292: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-31 17:54:21,294: INFO: common: created directory at: artifacts]
[2024-03-31 17:54:21,296: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-31 17:54:21,875: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-31 18:01:43,508: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-31 18:01:43,511: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-31 18:01:43,513: INFO: common: created directory at: artifacts]
[2024-03-31 18:01:43,514: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-03-31 18:01:43,891: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-03-31 18:02:01,783: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-31 18:02:01,786: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-31 18:02:01,788: INFO: common: created directory at: artifacts]
[2024-03-31 18:02:01,789: INFO: common: created directory at: artifacts\training]
[2024-03-31 22:13:13,479: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-31 22:13:13,482: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-31 22:13:13,483: INFO: common: created directory at: artifacts]
[2024-03-31 22:14:40,218: INFO: common: json file saved at: scores.json]
[2024-03-31 22:14:44,087: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-03-31 22:14:44,676: INFO: builder_impl: Assets written to: C:\Users\vaish\AppData\Local\Temp\tmp235_xcur\model\data\model\assets]
[2024-03-31 22:25:19,798: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-03-31 22:25:19,803: INFO: common: yaml file: params.yaml loaded successfully]
[2024-03-31 22:25:19,805: INFO: common: created directory at: artifacts]
[2024-03-31 22:26:45,979: INFO: common: json file saved at: scores.json]
[2024-03-31 22:26:50,423: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-03-31 22:26:51,022: INFO: builder_impl: Assets written to: C:\Users\vaish\AppData\Local\Temp\tmpsq7nbjb3\model\data\model\assets]
[2024-04-01 21:35:04,684: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 21:35:04,687: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 21:35:04,689: INFO: common: created directory at: artifacts]
[2024-04-01 21:35:04,691: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-01 21:35:04,692: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-01 21:35:38,197: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1oq-vYP-OzV85Nc4v6c-R5tYblyuGGJqy/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2024-04-01 21:37:39,147: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 21:37:39,150: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 21:37:39,152: INFO: common: created directory at: artifacts]
[2024-04-01 21:37:39,153: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-01 21:37:39,549: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-01 21:38:25,968: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 21:38:25,971: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 21:38:25,973: INFO: common: created directory at: artifacts]
[2024-04-01 21:38:25,975: INFO: common: created directory at: artifacts\training]
[2024-04-01 21:39:27,931: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 21:39:27,934: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 21:39:27,935: INFO: common: created directory at: artifacts]
[2024-04-01 21:39:27,937: INFO: common: created directory at: artifacts\training]
[2024-04-01 21:40:37,542: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 21:40:37,546: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 21:40:37,547: INFO: common: created directory at: artifacts]
[2024-04-01 21:40:37,549: INFO: common: created directory at: artifacts\training]
[2024-04-01 21:42:31,082: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 21:42:31,085: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 21:42:31,087: INFO: common: created directory at: artifacts]
[2024-04-01 21:42:31,088: INFO: common: created directory at: artifacts\training]
[2024-04-01 22:03:22,692: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 22:03:22,696: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 22:03:22,697: INFO: common: created directory at: artifacts]
[2024-04-01 22:03:22,699: INFO: common: created directory at: artifacts\training]
[2024-04-01 22:09:30,909: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 22:09:30,912: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 22:09:30,914: INFO: common: created directory at: artifacts]
[2024-04-01 22:09:30,915: INFO: common: created directory at: artifacts\training]
[2024-04-01 22:49:49,199: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 22:49:49,202: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 22:49:49,204: INFO: common: created directory at: artifacts]
[2024-04-01 22:49:49,205: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-01 22:49:49,207: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-01 22:49:55,197: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-01 22:50:05,772: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 22:50:05,776: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 22:50:05,778: INFO: common: created directory at: artifacts]
[2024-04-01 22:50:05,779: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-01 22:50:06,213: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-01 22:52:10,950: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-01 22:52:10,954: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-01 22:52:10,956: INFO: common: created directory at: artifacts]
[2024-04-01 22:52:10,957: INFO: common: created directory at: artifacts\training]
[2024-04-02 00:21:00,501: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 00:21:00,504: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 00:21:00,506: INFO: common: created directory at: artifacts]
[2024-04-02 00:22:26,144: INFO: common: json file saved at: scores.json]
[2024-04-02 00:22:29,593: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-04-02 00:22:30,220: INFO: builder_impl: Assets written to: C:\Users\vaish\AppData\Local\Temp\tmp9rl9t5o5\model\data\model\assets]
[2024-04-02 00:45:12,053: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-04-02 00:45:12,055: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 00:45:12,057: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 00:45:12,057: INFO: common: created directory at: artifacts]
[2024-04-02 00:45:12,057: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-02 00:45:12,058: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 00:45:18,496: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 00:45:18,919: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-04-02 00:45:18,919: INFO: main: *******************]
[2024-04-02 00:45:18,919: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-04-02 00:45:18,921: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 00:45:18,923: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 00:45:18,923: INFO: common: created directory at: artifacts]
[2024-04-02 00:45:18,923: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-02 00:45:19,321: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-02 00:45:19,528: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-04-02 00:45:19,528: INFO: main: *******************]
[2024-04-02 00:45:19,528: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-04-02 00:45:19,530: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 00:45:19,532: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 00:45:19,532: INFO: common: created directory at: artifacts]
[2024-04-02 00:45:19,533: INFO: common: created directory at: artifacts\training]
[2024-04-02 00:46:15,389: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-02 00:46:15,389: INFO: main: *******************]
[2024-04-02 00:46:15,389: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 00:46:15,391: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 00:46:15,393: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 00:46:15,393: INFO: common: created directory at: artifacts]
[2024-04-02 00:46:24,674: INFO: common: json file saved at: scores.json]
[2024-04-02 00:46:24,675: INFO: common: json file saved at: scores.json]
[2024-04-02 00:46:24,675: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-04-02 01:43:29,942: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-04-02 01:43:29,944: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 01:43:29,946: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 01:43:29,947: INFO: common: created directory at: artifacts]
[2024-04-02 01:43:29,947: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-02 01:43:29,947: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 01:43:35,524: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 01:43:36,100: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-04-02 01:43:36,100: INFO: main: *******************]
[2024-04-02 01:43:36,100: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-04-02 01:43:36,102: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 01:43:36,104: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 01:43:36,105: INFO: common: created directory at: artifacts]
[2024-04-02 01:43:36,105: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-02 01:43:36,509: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-02 01:43:36,713: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-04-02 01:43:36,713: INFO: main: *******************]
[2024-04-02 01:43:36,714: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-04-02 01:43:36,716: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 01:43:36,717: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 01:43:36,718: INFO: common: created directory at: artifacts]
[2024-04-02 01:43:36,718: INFO: common: created directory at: artifacts\training]
[2024-04-02 01:44:35,003: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-02 01:44:35,003: INFO: main: *******************]
[2024-04-02 01:44:35,003: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 01:44:35,006: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 01:44:35,007: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 01:44:35,008: INFO: common: created directory at: artifacts]
[2024-04-02 01:44:44,708: INFO: common: json file saved at: scores.json]
[2024-04-02 01:44:44,708: INFO: common: json file saved at: scores.json]
[2024-04-02 01:44:48,224: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-04-02 01:44:48,796: INFO: builder_impl: Assets written to: C:\Users\vaish\AppData\Local\Temp\tmp1f2wbfg9\model\data\model\assets]
[2024-04-02 01:45:34,431: ERROR: main: API request to endpoint /api/2.0/mlflow/registered-models/create failed with error code 403 != 200. Response body: '']
Traceback (most recent call last):
  File "main.py", line 50, in <module>
    model_evalution.main()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\stage_04_model_evaluation.py", line 20, in main
    evaluation.log_into_mlflow()
  File "d:\kidney-disease-classification\src\cnnClassifier\components\model_evaluation_mlflow.py", line 72, in log_into_mlflow
    mlflow.keras.log_model(self.model, "model", registered_model_name="VGG16Model")
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tensorflow\__init__.py", line 222, in log_model
    return Model.log(
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\models\model.py", line 562, in log
    mlflow.register_model(
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\_model_registry\fluent.py", line 77, in register_model
    raise e
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\_model_registry\fluent.py", line 68, in register_model
    create_model_response = client.create_registered_model(name)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\client.py", line 1743, in create_registered_model
    return self._get_registry_client().create_registered_model(name, tags, description)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\_model_registry\client.py", line 63, in create_registered_model
    return self.store.create_registered_model(name, tags, description)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\store\model_registry\rest_store.py", line 75, in create_registered_model
    response_proto = self._call_endpoint(CreateRegisteredModel, req_body)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\store\model_registry\base_rest_store.py", line 42, in _call_endpoint
    return call_endpoint(self.get_host_creds(), endpoint, method, json_body, response_proto)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\utils\rest_utils.py", line 281, in call_endpoint
    response = verify_rest_response(response, endpoint)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\utils\rest_utils.py", line 213, in verify_rest_response
    raise MlflowException(
mlflow.exceptions.MlflowException: API request to endpoint /api/2.0/mlflow/registered-models/create failed with error code 403 != 200. Response body: ''
[2024-04-02 14:20:50,545: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-04-02 14:20:50,548: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 14:20:50,549: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 14:20:50,550: INFO: common: created directory at: artifacts]
[2024-04-02 14:20:50,550: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-02 14:20:50,550: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 14:20:55,820: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 14:20:56,196: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-04-02 14:20:56,196: INFO: main: *******************]
[2024-04-02 14:20:56,196: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-04-02 14:20:56,198: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 14:20:56,200: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 14:20:56,200: INFO: common: created directory at: artifacts]
[2024-04-02 14:20:56,201: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-02 14:20:56,580: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-02 14:20:56,780: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-04-02 14:20:56,780: INFO: main: *******************]
[2024-04-02 14:20:56,781: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-04-02 14:20:56,783: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 14:20:56,784: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 14:20:56,785: INFO: common: created directory at: artifacts]
[2024-04-02 14:20:56,785: INFO: common: created directory at: artifacts\training]
[2024-04-02 14:21:52,806: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-02 14:21:52,806: INFO: main: *******************]
[2024-04-02 14:21:52,806: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 14:21:52,808: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 14:21:52,810: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 14:21:52,810: INFO: common: created directory at: artifacts]
[2024-04-02 14:22:02,161: INFO: common: json file saved at: scores.json]
[2024-04-02 14:22:02,161: INFO: common: json file saved at: scores.json]
[2024-04-02 14:22:05,561: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-04-02 14:22:06,162: INFO: builder_impl: Assets written to: C:\Users\vaish\AppData\Local\Temp\tmpjv7rpfsv\model\data\model\assets]
[2024-04-02 14:22:45,917: ERROR: main: API request to endpoint /api/2.0/mlflow/registered-models/create failed with error code 403 != 200. Response body: '']
Traceback (most recent call last):
  File "main.py", line 50, in <module>
    model_evalution.main()
  File "d:\kidney-disease-classification\src\cnnClassifier\pipeline\stage_04_model_evaluation.py", line 20, in main
    evaluation.log_into_mlflow()
  File "d:\kidney-disease-classification\src\cnnClassifier\components\model_evaluation_mlflow.py", line 72, in log_into_mlflow
    mlflow.keras.log_model(self.model, "model", registered_model_name="VGG16Model")
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tensorflow\__init__.py", line 222, in log_model
    return Model.log(
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\models\model.py", line 562, in log
    mlflow.register_model(
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\_model_registry\fluent.py", line 77, in register_model
    raise e
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\_model_registry\fluent.py", line 68, in register_model
    create_model_response = client.create_registered_model(name)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\client.py", line 1743, in create_registered_model
    return self._get_registry_client().create_registered_model(name, tags, description)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\tracking\_model_registry\client.py", line 63, in create_registered_model
    return self.store.create_registered_model(name, tags, description)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\store\model_registry\rest_store.py", line 75, in create_registered_model
    response_proto = self._call_endpoint(CreateRegisteredModel, req_body)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\store\model_registry\base_rest_store.py", line 42, in _call_endpoint
    return call_endpoint(self.get_host_creds(), endpoint, method, json_body, response_proto)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\utils\rest_utils.py", line 281, in call_endpoint
    response = verify_rest_response(response, endpoint)
  File "C:\Users\vaish\anaconda3\envs\kidney\lib\site-packages\mlflow\utils\rest_utils.py", line 213, in verify_rest_response
    raise MlflowException(
mlflow.exceptions.MlflowException: API request to endpoint /api/2.0/mlflow/registered-models/create failed with error code 403 != 200. Response body: ''
[2024-04-02 15:10:27,228: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 15:10:27,231: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 15:10:27,233: INFO: common: created directory at: artifacts]
[2024-04-02 15:10:37,228: INFO: common: json file saved at: scores.json]
[2024-04-02 15:10:41,149: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-04-02 15:10:41,747: INFO: builder_impl: Assets written to: C:\Users\vaish\AppData\Local\Temp\tmpg7fzy0tj\model\data\model\assets]
[2024-04-02 15:44:00,735: INFO: main: >>>>>> stage Data Ingestion Stage started <<<<<<]
[2024-04-02 15:44:00,738: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 15:44:00,739: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 15:44:00,740: INFO: common: created directory at: artifacts]
[2024-04-02 15:44:00,740: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-02 15:44:00,741: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 15:44:06,039: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 15:44:06,457: INFO: main: >>>>>> stage Data Ingestion Stage completed <<<<<<

x=========x]
[2024-04-02 15:44:06,457: INFO: main: *******************]
[2024-04-02 15:44:06,457: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2024-04-02 15:44:06,460: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 15:44:06,461: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 15:44:06,461: INFO: common: created directory at: artifacts]
[2024-04-02 15:44:06,462: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-02 15:44:06,854: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-02 15:44:07,049: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-04-02 15:44:07,050: INFO: main: *******************]
[2024-04-02 15:44:07,050: INFO: main: >>>>>> stage Training started <<<<<<]
[2024-04-02 15:44:07,052: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 15:44:07,054: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 15:44:07,054: INFO: common: created directory at: artifacts]
[2024-04-02 15:44:07,054: INFO: common: created directory at: artifacts\training]
[2024-04-02 15:45:02,957: INFO: main: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-02 15:45:02,957: INFO: main: *******************]
[2024-04-02 15:45:02,957: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 15:45:02,960: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 15:45:02,961: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 15:45:02,961: INFO: common: created directory at: artifacts]
[2024-04-02 15:45:12,448: INFO: common: json file saved at: scores.json]
[2024-04-02 15:45:12,448: INFO: common: json file saved at: scores.json]
[2024-04-02 15:45:15,995: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2024-04-02 15:45:16,564: INFO: builder_impl: Assets written to: C:\Users\vaish\AppData\Local\Temp\tmpilo3chyr\model\data\model\assets]
[2024-04-02 15:45:57,174: INFO: main: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
[2024-04-02 16:36:59,859: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-04-02 16:36:59,862: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 16:36:59,864: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 16:36:59,864: INFO: common: created directory at: artifacts]
[2024-04-02 16:36:59,865: INFO: common: created directory at: artifacts/data_ingestion]
[2024-04-02 16:36:59,865: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 16:37:06,616: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view into file artifacts/data_ingestion/data.zip]
[2024-04-02 16:37:07,036: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage completed <<<<<<

x=========x]
[2024-04-02 16:37:13,546: INFO: stage_02_prepare_base_model: *******************]
[2024-04-02 16:37:13,546: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model started <<<<<<]
[2024-04-02 16:37:13,548: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 16:37:13,551: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 16:37:13,551: INFO: common: created directory at: artifacts]
[2024-04-02 16:37:13,551: INFO: common: created directory at: artifacts/prepare_base_model]
[2024-04-02 16:37:13,949: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2024-04-02 16:37:14,182: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2024-04-02 16:37:18,962: INFO: stage_03_model_training: *******************]
[2024-04-02 16:37:18,962: INFO: stage_03_model_training: >>>>>> stage Training started <<<<<<]
[2024-04-02 16:37:18,965: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 16:37:18,967: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 16:37:18,967: INFO: common: created directory at: artifacts]
[2024-04-02 16:37:18,968: INFO: common: created directory at: artifacts\training]
[2024-04-02 16:38:17,937: INFO: stage_03_model_training: >>>>>> stage Training completed <<<<<<

x==========x]
[2024-04-02 16:38:23,626: INFO: stage_04_model_evaluation: *******************]
[2024-04-02 16:38:23,626: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2024-04-02 16:38:23,629: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-04-02 16:38:23,630: INFO: common: yaml file: params.yaml loaded successfully]
[2024-04-02 16:38:23,631: INFO: common: created directory at: artifacts]
[2024-04-02 16:38:33,663: INFO: common: json file saved at: scores.json]
[2024-04-02 16:38:33,663: INFO: common: json file saved at: scores.json]
[2024-04-02 16:38:33,664: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
